pkg/controller.v1/tensorflow/pod.go

```diff
@@ -349,6 +349,10 @@ func getContainerExitCode(pod *v1.Pod) int32 {
 // IsWorker0Completed return true if pod of worker0 succeeded and exited with 0
 func (tc *TFController) IsWorker0Completed(tfjob *tfv1.TFJob, replicas map[commonv1.ReplicaType]*commonv1.ReplicaSpec) (bool, error) {
 	worker0Completed := false
+	_, ok := replicas[tfv1.TFReplicaTypeWorker]
+	if !ok {
+		return true, nil
+	}
```

Comment:
Thanks for your contribution! :tada: :+1:

Can we add one unit test for it?

Reply:
I will add soon

Reply:
update a UT for IsWorker0Completed, but I am not sure use another *_test.go func is ok.